.changeset/better-tires-shave.md

```diff
@@ -0,0 +1,42 @@
+---
+"@gradio/client": minor
+"gradio": minor
+"gradio_client": minor
+---
+
+highlight:
+
+#### Automatically delete state when a user closes the browser tab
```

Comment:
Super nit: maybe we should consider rewording in light of the fact that the deletion is not immediate
```suggestion
#### Automatically delete state after user has disconnected from the webpage
```